### PR TITLE
Ayon server

### PR DESCRIPTION
**Pull request by martastain**

### CODE REVIEW

File: website/docs/admin_server_bootstrap_setup.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+
+## Configuration file
+
+Ayon server offers a convenient method for bootstrapping its basic settings 
+using a single JSON file. This enables you to quickly set up the server with 
+your desired configurations.
+
+By default, the server expects the configuration file to be located at `settings/template.json`. 
+The settings within this file will be applied each time you run the `make setup` command.
```
Comment (tokejepsen) [inline website/docs/admin_server_bootstrap_setup.md:19]:
```suggestion
The settings within this file will be applied each time you run the `make setup` (Unix) or `manage.ps1 setup` (Windows) command.
```

File: website/docs/admin_server_installation.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Installation
+
+1. Clone this repository to your local machine.
+2. Tweak the `docker-compose.yml` file according to your requirements.
+3. You may use the `.env` file to set environment variables (for example, for SSO configuration).
+4. On Windows, comment-out or delete the `- "/etc/localtime:/etc/localtime:ro"` line from the `docker-compose.yml` file.
+5. Install addons to the `addons` directory.
+6. Modify the default settings in the `settings/template.json` file.
+7. Run the stack using `docker compose up -d`
```
Comment (tokejepsen) [inline website/docs/admin_server_installation.md:25]:
```suggestion
7. Run the stack using `docker compose up -d` (Unix) or `docker-compse up -d` (Windows).
```
I know you already mention is above but cant hurt to be explicit.

**Review by tokejepsen — CHANGES_REQUESTED** [review]

File: website/docs/admin_server_installation.md (hunk 2 above)
Comment (mkolar) [reply website/docs/admin_server_installation.md:25]: Better to use linux and windows tabs for this. They can be synced in the whole page with the same tab setup as here https://github.com/ynput/ayon-documentation/blob/main/website/docs/dev_build.md?plain=1#L37-L252

File: website/docs/admin_server_installation.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Requirements
+
+- Docker with the compose plugin. To install the latest Docker, you can use this script: [https://get.docker.com](https://get.docker.com)
+- If you use a stand-alone `docker-compose` script instead of the compose plugin, make sure to use `docker-compose` wherever `docker compose` is used in this tutorial.
+- For production, using Linux is highly recommended. However, for evaluation purposes, Windows with WSL (Windows Subsystem for Linux) can be used.
+
+## Installation
+
+1. Clone this repository to your local machine.
```
Comment (mkolar) [inline website/docs/admin_server_installation.md:19]:
```suggestion
1. Clone [ayon-docker repository](https://github.com/ynput/ayon-docker) to your local machine.
```

File: website/docs/admin_server_bootstrap_setup.md (hunk 1 above)
Comment (martastain) [reply website/docs/admin_server_bootstrap_setup.md]: Fixed as @mkolar suggested (tabs)

File: website/docs/admin_server_provisioning.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+```markdown
+settings/
+  └── template.json
+```
+
+## Security Considerations
+
+Using the bootstrapped settings is particularly useful during the testing phase of your Ayon server. 
+However, once the server is properly set up and configured,
```
Comment (mkolar) [inline website/docs/admin_server_provisioning.md:28]: I'd wrap this into `:::caution` admonition

**Review by mkolar — APPROVED** [review]

**Pull request merged**